timezone_converter/comparison_view.py

```diff
@@ -1,10 +1,12 @@
 from datetime import datetime
 from datetime import timedelta
+from difflib import get_close_matches
 from typing import Iterable
 from typing import List
 from typing import Union
 
 import pytz
+from rich.console import Console
```

Comment:
This is not necessary, as we have a helper method to print rich objects, so no no need to import anything here.